QuangVu/exam1/exam1.css

```diff
@@ -0,0 +1,150 @@
+* {padding: 0px; margin: 0px;}
+
+body {
+    background-color: rgb(245, 244, 244);
```

Comment:
Please use hex color code.

Reply:
I've added .gitignore back. Check it again, please

Why can't we use RBG color????